NewSpring/Rock
# Added person analytics tracking to wistia embeds + new wistia button shortcode

## DESCRIPTION

This PR wires up the [logging of analytics to wistia that include person/content item/entity data](https://github.com/NewSpring/Rock/pull/1071/files#diff-c4f6221dc3c8f4b1cfe77c4b20f6b8b3R32) that gets re-linked back to Rock interactions when the wistia sync job runs.

It also adds a new lava shortcode that implements a button that launches a modal with a wistia video embedded in it.

### How do I test this PR?

1. Pull this branch down.
2. Create new `wistiaButton` shortcode in your environment's database - you can copy the lava/parameters defined here: https://brian-rock.newspring.cc/page/1176?lavaShortcodeId=98
3. Check the video embeds/buttons following pages/blocks:
- Sermon Detail Page Video Embed
- Editorial Detail Page Video Button (/stories)
- Page Builder Blocks (you'll need to add a video to these - Hero, Ad Unit, Side By Side)
- Network Music Resource Tutorial Videos: https://brian-new.newspringnetwork.com/resources/music/able

## TODO


## REVIEW


> The purpose of PR Review is to _improve the quality of the software._
